pipeline.yml

```diff
@@ -55,4 +51,25 @@ ci_test:
     format: "junit"
     pattern: "*.xml"
 
+soc2:
+  harmony:
+    hidden_email_list: dynamo_ws_access
+    third_party_lib_paths:
+      - "src\\GregClient\\bin\\Release\\net8.0\\"
```

Comment:
Isn't this `bin\\release\\net8.0\\`?

Reply:
no, in this sln there's only one project output we care about so it's directly in the project's bin folder.

Reply:
ah, it seems it was like that before and I accidentally removed the output path property from the csproj, let me refer that change.